### ISSUE TITLE
Ctrl+d remap in .vimrc is undetected

### ISSUE THREAD

**Issue by EdwinVanRooij** [VsVim/VsVim#2551]
Similar to my recent issue #2532. The issue there was that ctrl+i cannot be remapped in both Vim and VsVim, by design.

The issue here is that I can remap ctrl+d in Vim, but not in VsVim. 

The mapping `nnoremap <C-d>{d}` will delete the entire paragraph where the cursor is currently in (--> "delete around paragraph" in vim terms). 

In Vim this works, in VsVim the cursor still uses the default behavior, moving the cursor & screen a lot down.

Is this a known issue, expected behaviour, or a bug?

Thanks in advance!

**Comment by ricksladkey** [comment]
I'm not sure if there is a typo in your bug report, but the correct syntax for a mapping includes a space:

```
nnoremap <C-d> {d}
```

The fact that `<C-d>` scrolls by half a screen when you type it means that `<C-d>` is correctly set to "Handled by VsVim". So if you correct the mapping, I'm pretty sure that it will work.

**Comment by EdwinVanRooij** [comment]
Okay so I just did this:

Added `nnoremap <C-d> {d}` to my .vimrc and now it works in VsVim. I think I may know why this didn't work previously and it's pretty embarrassing. This is my current setup.

- I work on Windows, but I use Vim in the Ubuntu Linux subsystem as well. This means I have two different .vimrc files. One on the linux subsystem, and one on 'local' windows OS. 
- I changed the .vimrc at the location ~/.vimrc (on ubuntu linux subsystem) to include the `<C-d>` mapping. It worked there. 
- Then I changed the .vimrc at the location `/c/users/<name>/.vimrc`, which is the home directory on the 'local' windows OS. Then I tried it in VsVim, and it didn't work, so I posted the issue here.

Probably I actually did mistype the mapping in the windows .vimrc, I don't see any other reason why it would work now.

Thank you for pointing this out! Issue closed.
